login_page.php

```diff
@@ -104,11 +104,18 @@
 	}
 }
 
+# Redirect to plugin login page if applicable.
+if( !auth_can_use_standard_login( NO_USER ) ) {
```

Comment:
auth_can_use_standard_login receives a user_id parameter. Is this call right?
And, regarding the "can_use_standard_login", which currently is defined as a threshold, how do you know the user before it has logged?